### DISCUSSION TITLE
v0.11.2

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#427] [Releases]
## What's Changed
* Export selected result records by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/370
* Result comments by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/414
* Updated dependencies by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/415
* Local development env updates by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/416
* Release v0.11.2 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/426

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.11.1...v0.11.2

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.11.2'>v0.11.2</a>.</em>

**Comment by corasaniti** [comment]
Thanks!!! When you are planning InfluxDB1.x integration?

**Reply by p-jones** [reply]
Love your app!

Two minor suggestions:
- I cannot see how to change the default duration from the last 24 hrs to either week or month. I, for one, would prefer the default be the prior month. Would you consider adding this setting?
- minor typo on Settings | Notifications. It says "Notify on every **speetest** run" instead of speedtest.

**Reply by alexjustesen** [reply]
@corasaniti no date yet, got some released planned to fix a bunch of other things first. Identified a data issue so I'll obviously want to fix that first

@p-jones PRs are welcome to fix that if you have time
